<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExTestGUI</class>
 <widget class="QMainWindow" name="ExTestGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Config File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_config"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_config">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Data File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_data"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_data">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Voltage Reading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Lock-In</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nanovoltmeter</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>No. of Data Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_dataPoints"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max Field (kG)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_maxF"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min Field (kG)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_minF"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PlotWidget" name="plot_widget"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="button_go">
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_abort">
        <property name="text">
         <string>Abort Measurement</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
